### ISSUE TITLE
The output packages don't have locPak files

### ISSUE THREAD

**Issue by KaneGreen** [hallidave/landing-monitor#1]
I plan to contribute a localization.
So I'm trying to build my own version for testing.
I'm currently using SU 11, SDK 0.20.4, Node.js v18.12.1.

After I run `npm run package`, the project built successfully and was saved in `project/Packages/hallidave-tool-landing`.
However, there are no locPak files (eg `en-US.locPak`) in this directory.

Besides, after entering the game, this add-on loads successfully, but doesn't display the correct text no matter what the game language is set, just showing text starting with `LANDING.`.

**Comment by hallidave** [comment]
I think they changed localization in the latest SDK for SU11. I saw something in the release notes about it. I haven't had a chance to look at it yet.

**Comment by hallidave** [comment]
I just pushed an update to support the new way that localization is done for the SU11 SDK. There is a new tool in the SDK called MSFS Localization Manager (`<MSFS SDK Path>\Tools\MSFS_Localization\MSFSLocalizationManager.exe`) for editing localization translations. I updated the "Localization" heading in the [README.md](https://github.com/hallidave/landing-monitor#localization) file with more details.
